.travis.yml

```diff
@@ -1,12 +1,62 @@
 language: dart
-sudo: false
 dart:
-  - stable
   - dev
+
 install:
   - gem install coveralls-lcov
-script: ./tool/travis.sh
+
+# Gives more resources on Travis (8GB Ram, 2 CPUs).
```

Comment:
From what I've been told by Travis support, this gives a more powerful machine but builds take longer to start. Let's see how it goes. Probably doesn't matter for this repo.